### PR TITLE
Adding Oracle Linux 8 image

### PR DESCRIPTION
**Pull request by ib1984**
I've created an Oracle Linux 8 image based on CentOS 8 image with some adjustments. Enjoy!

### CODE REVIEW

File: oracle/files/rc-local.conf [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+[Service]
+Type=forking
+ExecStart=/etc/rc.local start
+TimeoutSec=0
+StandardOutput=tty
+RemainAfterExit=yes
+SysVStartPriority=99
+
+[Install]
+WantedBy=multi-user.target
```
Comment (oneumyvakin) [inline oracle/files/rc-local.conf]: Please add new line to the end of the file

File: oracle/files/rc.local [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+clean_up(){
+  rm -rf /etc/rc.local
+  rm -rf /etc/rc.d/rc.local
+
+  systemctl disable rc-local
+  systemctl stop rc-local
+	rm -rf /etc/systemd/system/rc-local.service
+	rm -rf /etc/subgid.lock
```
Comment (oneumyvakin) [inline oracle/files/rc.local:18]: Please use the same indents in whole file, like on line 4 for example.

**Review by oneumyvakin — CHANGES_REQUESTED** [review]

File: oracle/files/rc.local (hunk 2 above)
Comment (ib1984) [reply oracle/files/rc.local:18]: This one comes from the original **centos/files/rc.local**. Fixed.

**Comment by ib1984** [comment]
Fixed whitespaces.

**Review by oneumyvakin — APPROVED** [review]

**Pull request merged**